metecho/api/filters.py

```diff
@@ -28,7 +28,7 @@ class TaskFilter(filters.FilterSet):
 
     class Meta:
         model = Task
-        fields = ("epic", "slug")
+        fields = ("epic", "project", "slug")
```

Comment:
Oh, huh. This made me realize that the `view-all-tasks` PR wasn't actually filtering tasks by project. Fixed in https://github.com/oddbird/Metecho/commit/e67815b88e49f8c607886c1111184852384c6bc8, but we'll need to resolve that conflict when merging into this PR. (Ideally tasks could be filtered by `project`, which would include matches for both `task.epic.project` and `task.project`.)

Reply:
95961f60424ed49568065913d1f333e8bd8cd67d should take care of it